docs/common-bugs.rst

```diff
@@ -344,7 +344,7 @@ Automatic installation in Live OS
 
 :Issue: The reporter would like to run a kickstart installation in Live OS.
```

Comment:
The reporter mentioned that it was difficult to find a solution for the message: `Kickstart is not supported on live installs. This installation will continue interactively.` Could you maybe document this message in the issue description?

Reply:
Yes, but I'll use the new one.

Reply:
Used both - f33 is still "active", and f34 will be, so the old message will appear more often for at least a year.